contents/docs/libraries.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Libraries
+sidebar: Docs
+showTitle: true
+---
+
+PostHog provides multiple libraries to send events from your app to your PostHog instance.
+
+We have put a significant effort into providing as many of these libraries as possible, in order to make your experience better.
+
+Our goal is for users to be able to integrate PostHog with their language and framework of choosing, without having to adapt their stack.
+
+## Libraries
+
+**All of PostHog's libraries support event capturing and user identification.**
+
+However, certain features like session recording are only available with our JavaScript library. Refer to the table below for an overview of the functionality in each of our libraries.
+
+|                     Library                     |  Type  |  Maintainer  | Autocapture | Session Recording | Feature Flags |
```

Comment:
For feature flags, this might be too much manual syncing, but we could also replace "X" with "Requested" and link to the relevant issues, e.g:
- https://github.com/PostHog/posthog-ios/issues/14
-

Reply:
Hmm "Requested" sounds a bit bad to me - as in we haven't gotten to it.

Plus we get requests on Slack and calls all the time

Reply:
yeah, totally ok to skip this